packages/cycle-scripts/package.json

```diff
@@ -30,12 +30,16 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
```

Comment:
You mean if we keep that dependency? Is that something we really wanna do then?
Tbh, now that I see how it works, is not such a big deal achieve the same thing without block, as mainly is env, devserver, and different plugins for typescrpt. I think you can achieve this easily within a plain webpack config object. What make you so vocal about this? I'm probably missing something as I cannot see such an added value, apart of taking ownership of important dependencies like loaders&co away from you...i must be wrong though...